oz/Guest.py

```diff
@@ -481,10 +482,8 @@ def _generate_xml(self, bootdev, installdev, kernel=None, initrd=None,
         if initrd:
             oz.ozutil.lxml_subelement(osNode, "initrd", initrd)
         if cmdline:
-            if self.tdl.arch == "armv7l":
-                cmdline += " console=ttyAMA0"
             oz.ozutil.lxml_subelement(osNode, "cmdline", cmdline)
-        if self.tdl.arch == "aarch64":
+        if self.tdl.arch in ["x86_64", "aarch64", "armv7l"]:
```

Comment:
this means that all x86_64 will default to UEFI based installs. This needs to be configurable. See https://pagure.io/releng/issue/8197